docs/workflows.rst

```diff
@@ -225,20 +227,47 @@ BOLD reference image estimation
     from fmriprep.workflows.bold import init_bold_reference_wf
     wf = init_bold_reference_wf(omp_nthreads=1)
 
-This workflow estimates a reference image for a BOLD series, which is used to
-:ref:`estimate head motion <bold_hmc>` and :ref:`register BOLD series to T1w
-<bold_reg>`,
-and performs skull-stripping and contrast enhancement.
+This workflow estimates a reference image for a
+:abbr:`BOLD (blood-oxygen level-dependent)` series.
 If T1-saturation effects ("dummy scans" or non-steady state volumes) are
 detected they are used as reference due to their superior tissue contrast.
 Otherwise a median of motion corrected subset of volumes is used.
-
-Skull-stripping of the reference image is performed using Nilearn.
+The reference image is used to calculate a brain mask for the
+:abbr:`BOLD (blood-oxygen level-dependent)` signal using
+`Nilearn <http://nilearn.github.io/>`_.
+Further, the reference is fed to the :ref:`head-motion estimation
+workflow <bold_hmc>` and the :ref:`registration workflow to map
+BOLD series into the T1w image of the same subject <bold_reg>`.
 
 .. figure:: _static/brainextraction.svg
     :scale: 100%
 
-    Brain extraction (BET).
+    Calculation of a brain mask from the BOLD series.
+
+.. _bold_hmc:
+
+Head-motion estimation
+~~~~~~~~~~~~~~~~~~~~~~
+:mod:`fmriprep.workflows.bold.hmc.init_bold_hmc_wf`
+
+.. workflow::
+    :graph2use: colored
+    :simple_form: yes
+
+    from fmriprep.workflows.bold import init_bold_hmc_wf
+    wf = init_bold_hmc_wf(
+        mem_gb=1,
+        omp_nthreads=1)
+
+Using the previously :ref:`estimated reference scan <bold_ref>`,
+FSL ``mcflirt`` is used to estimate head-motion.
+As a result, one rigid-body transform w.r.t. the reference image
+is written for each :abbr:`BOLD (blood-oxygen level-dependent)`
+time-step. Additionally, a list of 6-parameters (three rotations,
+three translations) per time-step is written and fed to the
+:ref:`confounds workflow <bold_confounds>`.
+Motion parameters are therefore calculated before any time-domain
```

Comment:
The above doesn't read as an argument that would lead naturally to the conclusion. "Therefore" made me look back to see if an argument was being made. It's possible that its use in the confounds workflow is the justifying statement, but, if so, I think I might rewrite this last sentence as:

"To improve confound estimation, we calculate motion parameters before any time-domain
filtering (i.e. slice-timing correction), as recommended in [Power2017]_."

(Replace "improve confound estimation" with a more precise justification.